src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java

```diff
@@ -0,0 +1,24 @@
+package edu.cornell.kfs.gl.service.impl.fixture;
+
+public enum EmailAdressTestValue {
+    THREE_PART_DOMAIN_GOOD("ww5@a.b.c.org", true),
+    ALPHANUMERIC_DOMAIN_DOT_ADDRESS_GOOD("something.else@a2.com", true),
+    ALPHANUMERIC_DOMAIN_UNDERSCORE_ADDRESS_GOOD("something_else@a2.com", true),
+    ALPHANUMERIC_DOMAIN_HYPHEN_ADDRESS_GOOD("something-else@a2.com", true),
+    UNDERSCORE_ADDRESS_GOOD("something_else@something.else.com", true),
+    HYPHEN_DOCMAIN_GOOD("something-else@et-tu.com", true),
+    NO_ADDRESS_BAD("@a.b.c.org", false),
+    NO_DOMAIN_BAD("a", false),
+    SPACE_BAD(" @a", false),
+    HYPHEN_ADDRESS_BAD("\"-@a.org\"", false),
+    UNDERSCOREW_DOMAIN_BAD("someone@foo_bar.com", false);
```

Comment:
I think there is a typo here also. Should it be UNDERSCORE_DOMAIN_BAD ?

Reply:
ok